crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -165,21 +165,43 @@ impl ComponentSparseSet {
     }
 
     #[inline]
-    pub fn get_with_ticks(&self, entity: Entity) -> Option<(Ptr<'_>, &UnsafeCell<ComponentTicks>)> {
+    pub fn get_with_ticks(
+        &self,
+        entity: Entity,
+    ) -> Option<(Ptr<'_>, &UnsafeCell<Tick>, &UnsafeCell<Tick>)> {
```

Comment:
Yeah we *really* want to newtype that pair if we can: exposing a tuple with two critically different complex objects of the same type is not nice.

Reply:
Done.